packages/react/src/NavList/NavList.stories.tsx

```diff
@@ -225,7 +225,32 @@ export const WithInactiveItems: StoryFn = () => (
   </PageLayout>
 )
 
-export const WithGroup = () => (
+export const WithLoadingItems: StoryFn = () => (
+  <PageLayout>
+    <PageLayout.Pane position="start">
+      <NavList>
+        <NavList.Item href="#" loading>
```

Comment:
Now that we don't have `loading` for links, should this prop & story be removed from `NavList`? I noticed that this story wasn't working, and I think it's because all `NavList.Item`(s) utilize `ActionList.LinkItem` by default.

Reply:
Thanks - I missed this one.